topics/dev/tutorials/community-tool-table/tutorial.md

```diff
@@ -106,41 +106,31 @@ Once the Pull Request is merged, the interactive table will be created and made
 
 The interactive table will contain all the tools associated with the ToolShed categories that you selected. However, not all of these tools might be interesting for your community. 
 
-Galaxy Tool Metadata Extractor provides 2 optional filters for tools:
-- A list of tools to exclude
-- A list of tools that should definitely be kept.
+Galaxy Tool Metadata Extractor provides an optional filter for tools, that can be defined by the community curator (maybe that is you !).
 
-## Add a list of tools to exclude
+## Add a tools status list 
```

Comment:
We could maybe remove this section and make them adding `tools_status.tsv` only in "Review all tools in your table" section. wdyt?

Reply:
I had the same idea, its probably cleaner ... will adapt.